src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -109,28 +111,40 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             {
                 extensionsPath = Path.Combine(_rootScriptPath, "bin");
```

Comment:
nit: Should we rename this `extensionsJsonPath`? Since the extensions may not actually be here?

Reply:
Went ahead with that. Let me know if you have other ideas.